integration

```diff
@@ -528,6 +528,7 @@
   "kpoppel/homeassistant-eforsyning",
   "kpoppel/homeassistant-novafos",
   "krahabb/meross_lan",
+  "krasnoukhov/homeassistant-oncharger",
```

Comment:
You should really not log out credentials https://github.com/krasnoukhov/homeassistant-oncharger/blob/main/custom_components/oncharger/__init__.py#L28
They will accidentally be posted to github issues, which is not great.